### PR TITLE
Get file content

### PR DESCRIPTION
**Pull request by mludvik**
Hi and thanks for this extension :cat: I would like to create pdf and then save it using your another [file-storage](https://github.com/yii2tech/file-storage) extension. Is there already nice way how to do it?

If not, I am proposing this simple new method `getContent`, so that you can do...

```
$content = $html2pdf->convert($html)->getContent();
$bucket->saveFileContent('output.pdf', $content);
```

Thanks for your time :smiley:

### CODE REVIEW

File: src/TempFile.php [hunk 1]
```diff
@@ -102,6 +102,15 @@ public function delete()
         return false;
     }
 
+    /**
+     * Gets content of this file.
```
Comment (klimov-paul) [inline src/TempFile.php:106]: `@since 1.0.4` should be added to the docblock

File: tests/TempFileTest.php [hunk 2]
```diff
@@ -67,4 +67,15 @@ public function testMove()
         $this->assertTrue(file_exists($destinationFileName));
         $this->assertFalse(file_exists($fileName));
     }
+
+    public function testGetContent() {
```
Comment (klimov-paul) [inline tests/TempFileTest.php]: Open bracket should be moved to the new line.

**Review by klimov-paul — COMMENTED** [review]
CHANGELOG entry should be added.

**Comment by mludvik** [comment]
Thanks for feedback @klimov-paul . All issues fixed.

**Review by klimov-paul — APPROVED** [review]

**Comment by klimov-paul** [comment]
Thank you.

**Pull request merged**